spring-boot/admin/src/main/java/eu/coatrack/admin/security/GatewayAuthFilter.java

```diff
@@ -44,24 +45,19 @@
  *
  * @author gr-hovest
  */
-public class GatewayAuthFilter extends GenericFilterBean {
+public class GatewayAuthFilter extends OncePerRequestFilter {
 
     private static final Logger log = LoggerFactory.getLogger(GatewayAuthFilter.class);
 
     @Override
-    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain) throws IOException, ServletException {
+    protected void doFilterInternal(HttpServletRequest servletRequest, HttpServletResponse servletResponse, FilterChain filterChain) throws ServletException, IOException {
```

Comment:
The method is too long. Additionally there are three levels of indentation, one indentation (e.g. one if statement with any nests) per method would be ideal. I think you should split that logic to multiple smaller functions to leave it tidy and readable.

Reply:
what do you mean with method is too long, this method already exists on the OncePerRequestFilter, so I am just overriding it to put the config we need

Reply:
My impression is that this is a discussion about "programming best practices". However, this method is not relevant for the pull request as far as I can see, so if you think a change is required, please add a new issue (with low priority).